### DISCUSSION TITLE
Use with vanilla Js?

### DISCUSSION THREAD

**Discussion by pknotfound** [BradyDouthit/cytoscape-html#3] [Q&A] [answered]
Is there a way to use with Vanilla js like cytoscape?

**Comment by BradyDouthit** [comment, marked as answer]
> Is there a way to use with Vanilla js like cytoscape?

Hey, sorry for the late reply. Hopefully this is still useful to you. 

Starting with v0.5.0 you should be able to use it with Vanilla JS but you would have to download the source code yourself and import it as `cytoscape-html` is currently not in a CDN or anything.

**Reply by tark-hidden** [reply]
Is it possible to build a production version without external .ts files and eval functions in cytoscape-html.js?..
